python/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+
+services:
+  ddtracepy-dev:
+    image: docker.pkg.github.com/datadog/dd-trace-ci/ddtracepy-dev:1.0
```

Comment:
can probably just do `dd-trace-ci/python-dev:latest`

PHP does version specific images, but we have an all-in-one image.